libs/features/personalization/personalization.js

```diff
@@ -154,7 +156,9 @@ const COMMANDS = {
 };
 
 function checkSelectorType(selector) {
-  return selector?.includes('/fragments/') ? 'fragment' : 'css';
+  if (selector?.includes('/fragments/')) return 'fragment';
+  if (selector?.startsWith(GLOBAL_NAV_SELECTOR)) return GLOBAL_NAV_SELECTOR;
```

Comment:
Should we account for possible whitespace before "global-navigation"?
```suggestion
  if (selector?.trim().startsWith(GLOBAL_NAV_SELECTOR)) return GLOBAL_NAV_SELECTOR;
```